tests/unit_tests/unit_test_world_builder.cc

```diff
@@ -294,7 +294,16 @@ TEST_CASE("WorldBuilder Point: Testing initialize and operators")
   CHECK_THROWS_WITH(Point<2>(1,2,3,cartesian),Contains("Can't use the 3d constructor in 2d."));
   CHECK_THROWS_WITH(Point<3>(1,2,cartesian),Contains("Can't use the 2d constructor in 3d."));
 
-  ApprovalTests::Approvals::verifyAll("TITLE", approval_tests);
+
+  std::vector<std::string> approvals;
+  for (auto&& value : approval_tests)
+    {
+      std::stringstream s;
+      s << value.first << " - " << value.second;
```

Comment:
This leads to a lot of `-` signs without a preceding name, which looks a bit like a minus sign and messes up your existing approval results. What do you think about the following?
```suggestion
      s << value.first << (value.first != "") ? " - " : "" << value.second;
```